src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -32,6 +32,7 @@
 
     // Independent Ui parts residing in this Ui container
     private StudentListPanel studentListPanel;
+    private StudentListPanel sessionListPanel;
```

Comment:
Should we create another SessionListPanel class for this instead? Don't think StudentListPanel class should consist of Session objects.